app/controllers/idv/sessions_controller.rb

```diff
@@ -40,18 +35,17 @@ def cancel_in_person_enrollment_if_exists
         cancel_stale_establishing_enrollments_for_user(current_user)
     end
 
-    def clear_session
-      user_session['idv/doc_auth'] = {}
-      user_session['idv/in_person'] = {}
-      idv_session.clear
-      Pii::Cacher.new(current_user, user_session).delete
+    def cancel_inherited_proofing_if_exists
+      return if !inherited_proofing?
+      # LG-7128: Implement Inherited Proofing analytics here.
+      # analytics.inherited_proofing_start_over(
+      #   step: location_params[:step],
+      #   location: location_params[:location],
+      # )
```

Comment:
also one more idea, I think we don't need this separate `idv_start_over` since we should be able to pass a flow/param/analytics_ids param to the existing `idv_start_over` event

Reply:
Ok, that's good to know.